README.md

```diff
@@ -17,20 +25,9 @@ git clone https://github.com/hangoutsidesf/parklet-crawl.git crawler
 cd crawler
 ```
 
-2. Install Dependencies
-```
-npm install
-```
-
-3. Build
-```
-npm run build
-```
-> will compile src to /dist
-
-4. Collect Data
-```
-npm start
+2. Start Production Container
+```bash
+npm run deploy:prod
```

Comment:
The user will need to setup a `.env` file with a custom `YELP_API_KEY` before running the crawler. I guess since we're refactoring the README here, might as well add that small detail.